lib/assigner/include/zkevm_word.hpp

```diff
@@ -183,6 +189,11 @@ namespace nil {
                 return static_cast<uint64_t>(value[i]);
             }
 
+            const value_type &get_value_type() const
```

Comment:
`get_value`
`get_value_type` sounds like returns the type of value

Reply:
Agree, fixed